enterprise/api/v1/views/enterprise_group.py

```diff
@@ -157,18 +172,34 @@ def assign_learners(self, request, group_uuid):
             customer = group.enterprise_customer
         except models.EnterpriseGroup.DoesNotExist as exc:
             raise Http404 from exc
-        if requested_emails := request.POST.dict().get('learner_emails'):
+
+        if requested_emails := request.data.get('learner_emails'):
+            request_data = {}
+            if act_by_date := request.data.get('act_by_date'):
+                request_data['act_by_date'] = act_by_date
+            if catalog_uuid := request.data.get('catalog_uuid'):
+                request_data['catalog_uuid'] = catalog_uuid
+            request_data['learner_emails'] = requested_emails.split(',')
```

Comment:
I have the same question as Kira - why can't we pass `request.data` directly into this serializer for validation?  This is a POST endpoint, so the caller can/should provide some JSON data as the request body, which means they can provide a list instead of a comma-delimited string.

Reply:
Thanks for this callout. I updated this to `request.data.getlist` for the emails since `request.data.get` only returns the last item in the list per the specs.